translations/en.json

```diff
@@ -58,7 +58,8 @@
   "analysisSettings": "Analysis Settings",
   "analytics": "Analytics",
   "analyzing": "Analyzing",
-  "anyDevice": "Any Device",
+  "android": "Android",
+  "anyDevice": "Use any available device with any OS version",
```

Comment:
key and value don't match

Reply:
Update to `anyAvailableDeviceWithAnyOS`.